gluten-core/src/main/scala/io/glutenproject/execution/WindowExecTransformer.scala

```diff
@@ -80,6 +81,26 @@ case class WindowExecTransformer(
 
   override def outputPartitioning: Partitioning = child.outputPartitioning
 
+  def genWindowParametersBuilder(): com.google.protobuf.Any.Builder = {
+    // Start with "WindowParameters:"
+    val windowParametersStr = new StringBuffer("WindowParameters:")
+    // isStreaming: 1 for streaming, 0 for sort
+    val isStreaming: Int =
+      if (GlutenConfig.getConf.veloxColumnarWindowType.equals("streaming")) 1 else 0
+
+    windowParametersStr
+      .append("isStreaming=")
+      .append(isStreaming)
+      .append("\n")
+    val message = StringValue
+      .newBuilder()
+      .setValue(windowParametersStr.toString)
+      .build()
+    com.google.protobuf.Any.newBuilder
+      .setValue(message.toByteString)
+      .setTypeUrl("/google.protobuf.StringValue")
+  }
+
```

Comment:
this part looks confusing.. could we use local variable to capture config and define a window type in substrait?

Reply:
@Yohahaha Thanks for your review. Previously, in order to minimize changes to the substrait, we placed these configs within the extension, such as the isSMJ parameter mentioned [here](https://github.com/oap-project/gluten/blob/6c38a76b6ec5e4c398a65af9a8f17d3da33a5107/gluten-core/src/main/scala/io/glutenproject/execution/SortMergeJoinExecTransformer.scala#L179). This was done because the substrait community is not very receptive to our customization of parameters specific to Velox.